client/src/store/reducer/employeeTableReducer.js

```diff
@@ -0,0 +1,169 @@
+const initalState = {
+  employeeData: undefined,
+  filteredEmployeeData: undefined,
+  filters: {
+    department: {},
+    office: {},
+    name: '',
+  },
+  recipients: [],
+  page: 0,
+  rowsPerPage: 10,
+};
+
+const toggleSingleRecipient = (array, item) => {
+  const itemIndex = array.indexOf(item);
+  if (itemIndex === -1) {
```

Comment:
you can do `if (array.includes(item))` and avoid needing to define `itemIndex` altogether. Sim. for `toggleRecipients`